youtube_dl/postprocessor/ffmpeg.py

```diff
@@ -429,20 +431,43 @@ def add(meta_list, info_list=None):
 
         filename = info['filepath']
         temp_filename = prepend_extension(filename, 'temp')
+        in_filenames = [filename]
+        tmp_filenames = []
+
+        options = ['-map_metadata', '1']
```

Comment:
it might be a good idea to keep the original metadata even when there is no `chapters`.